Detector/DetFCChhTrackerTkLayout/src/BeamTube_geo.cpp

```diff
@@ -0,0 +1,35 @@
+
```

Comment:
There seems to be a CMake Configuration issue in this version of ACTS, resulting in warnings from all subfolders that depend on DetCommon if I move it. I propose to leave it here.